src/Components/RoutingMenu/RoutingMenu.scss

```diff
@@ -2,13 +2,30 @@
 @import url('https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap');
 
 .rd-routing-menu {
+  width: 100%;
+  display: flex;
   z-index: 1;
-  position: absolute;
-  width: 500px;
-  max-width: 500px;
-  flex-grow: 1;
-  left: 10px;
-  top: 10px;
+
+  > div {
+    z-index: 1;
+    margin-top: 32px;
+    flex-grow: 1;
+  }
+
+  .rd-routing-menu-toggle {
+    background-color: white;
+    height: 32px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border: none;
+    padding: 0;
+    box-shadow: 0 0 5px #555;
+    color: #555;
+    width: 100%;
+    position: absolute;
+    z-index: 1;
```

Comment:
Personally I think the toggle btn should have a higher z-Index than the Routing-Menu, so the menu will slide out underneath the button on mobile view. Adjust if you agree...

Reply:
I also just noticed that the + button is partly obscured on desktop view:

![routing](https://user-images.githubusercontent.com/25749955/135219094-39530081-3f93-48f8-b45a-2aaf97589875.png)